internal/backend/local/backend_local.go

```diff
@@ -80,6 +80,8 @@ func (b *Local) localRun(ctx context.Context, op *backend.Operation) (*backend.L
 	coreOpts.UIInput = op.UIIn
 	coreOpts.Hooks = op.Hooks
 	coreOpts.Encryption = op.Encryption
+	// TODO andrei should we do this in remote/backend too?
```

Comment:
I would not extend those due to our limited ability to test those backends out. Also, implementations of remote / cloud backend APIs might potentially need to support this flag as well. I would treat it as a separate issue if someone from the community asks for it.